public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx

```diff
@@ -71,7 +88,14 @@ export const LokiVariableQueryEditor: FC<Props> = ({ onChange, query }) => {
       {type === QueryType.LabelValues && (
         <>
           <InlineField label="Label" labelWidth={20}>
-            <Input type="text" aria-label="Label" value={label} onChange={onLabelChange} onBlur={handleBlur} />
+            <Select
+              aria-label="Label"
+              onChange={onLabelChange}
+              onBlur={handleBlur}
+              value={label}
+              options={labelOptions}
+              width={16}
```

Comment:
This should also have `allowCustomValue={true}` because in Loki, you can create variable (for example to fetch label names) and then use that variable in the next variable (to fetch values for any selected value). So we should let users create the variable. Or even better - offer it?

Reply:
Good idea.

Reply:
Done!